.github/scripts/.helm-tests/lifecycle-with-certs/result.yaml

```diff
@@ -12290,6 +12290,7 @@ spec:
         runAsNonRoot: true
       serviceAccountName: lifecycle-operator
       terminationGracePeriodSeconds: 10
+      hostNetwork: false
```

Comment:
Thank you for your contribution! :) Would it be possible to enable the `hostNetwork` for at least one helm test?

Reply:
Thanks for the review on my first contribution.
I've added the parameter to the Deployment. Besides that I've also added the enabled `hostNetwork` in one of the `helm-tests`.

Reply:
Thank you!